builtin/bisect--helper.c

```diff
@@ -1015,6 +1130,12 @@ int cmd_bisect__helper(int argc, const char **argv, const char *prefix)
 			die(_("--bisect-log requires 0 arguments"));
 		res = bisect_log();
 		break;
+	case BISECT_REPLAY:
+		if (argc != 1)
+			die("No logfile given");
```

Comment:
Shouldn't we `return error(...)` here also?

Reply:
Also it looks like the string should be translated.

Reply:
(And start with a lower-case letter ;-))